src/core-tags/migrate/util/import-tag.js

```diff
@@ -19,13 +20,27 @@ module.exports = function importTag(importPath, context) {
     const requestedTagName = camelCase((match && match[1]) || "Template", {
         pascalCase: true
     }).replace(/^[^$A-Z_]|[^0-9A-Z_$]/gi, "_");
+    const symbols = [];
+    const walker = context.createWalker({
+        enter(node) {
+            // Try to fetch all symbols and then put them in an array
+            // We will check if any symbols or imports have matching items
+            // in order to prevent variable collisions
+            if (node.type === "HtmlElement" && node.tagName === "import") {
+                symbols.push(node.tagString);
+            } else if (node.type === "Scriptlet") {
+                symbols.push(node.code);
+            }
+        }
+    });
+    walker.walk(context.root);
 
     let identifier = requestedTagName;
     let i = 1;
 
     while (
         migrateImports.identifiers[identifier] ||
```

Comment:
This check may be redundant now.  Previously we were looking at the original source, but now we we're checking all imports and scriptlets from the current ast.  When a new import identifier is added, the ast should reflect this the next time this check is run.

Reply:
I removed it and the tests are timing out (looping indefinitely), I think this also helps if it has already been loaded it does not duplicate that line of code.